garcon/decider.py

```diff
@@ -61,6 +61,22 @@ def get_history(self, pool):
         # Remove all the events that are related to decisions and only.
         return [e for e in events if not e['eventType'].startswith('Decision')]
 
+    def get_workflow_execution_info(self, pool):
+        """Get the workflow execution info from a given pool if it exists.
+
+        Args:
+            pool (object): The pool object (see AWS SWF for details.)
+        Return:
+            `dict`: Workflow execution info including workflowId and runId.
+        """
+
+        if 'workflowExecution' in pool:
+            return pool['workflowExecution'] if 'workflowId' \
```

Comment:
This looks great. 

* Can you add a few things: create a dictionary that hydrate the context with `execution.workflow_id` and `execution.run_id`? 
* And can you write a quick test for `get_workflow_execution_into`? You will just do two tests: when you get the right information and when you don't. It should be fairly straightforward.